app/src/main/java/com/x8bit/bitwarden/data/platform/manager/BiometricsEncryptionManagerImpl.kt

```diff
@@ -89,7 +94,10 @@ class BiometricsEncryptionManagerImpl(
 
     /**
      * Generates a [SecretKey] from which the [Cipher] will be generated.
+     *
+     * @throws ProviderException if [KeyGenerator] is unable to generate a key.
      */
+    @Throws(ProviderException::class)
```

Comment:
We generally don't do this, it's kinda a Java thing. How strongly do you feel about it?